src/elements/Header/_Header.js

```diff
@@ -1,17 +1,22 @@
-import _ from 'lodash'
-import React, { Component, PropTypes } from 'react'
-import classNames from 'classnames'
+import cx from 'classnames'
```

Comment:
Not relevant to this line... I wonder if it would make sense to make this a pure component and not re-render unless props change. Since it's such a simple (and common) element it might be a nice future feature (or option) to do prop checks to prevent unneeded re-renders.